boot/cypress/MCUBootApp/libs.mk

```diff
@@ -0,0 +1,90 @@
+################################################################################

```

Comment:
This file has DOS line endings. It should be Unixy endings to be consistent with the rest of the repo.